install_config/build_defaults_overrides.adoc

```diff
@@ -0,0 +1,109 @@
+= Configuring Global Build Defaults and Overrides
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Developers can define settings in specific build configurations within their
+projects, such as
+link:../dev_guide/builds.html#using-a-proxy-for-git-cloning[configuring a proxy
+for Git cloning]. Rather than requiring developers to define certain settings in
+each of their build configurations, cluster administrators can use admission
+control plug-ins to configure global build defaults and overrides that
+automatically populate these settings into any build configuration.
+
+The `*BuildDefaults*` admission control plug-in allows an administrator to set
+global defaults for settings such as the Git HTTP and HTTPS proxy, and default
+environment variables. These defaults do not overwrite values that have been
+configured for a specific build. However, if those values are not present on the
+build definition, they are set to the default value.
+
+The `*BuildOverrides*` admission control plug-in allows overriding a setting in
```

Comment:
s/allows overriding a setting/allows the overriding of a setting 
Alternatively (maybe better):
"allows a setting in a build to be overridden"

Maybe? Just seems like awkward phrasing to me.